src/scss-styles/components/_buttons.scss

```diff
@@ -0,0 +1,7 @@
+/*Button Theme Primary */
```

Comment:
Why this file is empty committed? use this file for button styles

Reply:
button styles updated in button common file